src/storage.ts

```diff
@@ -754,19 +1052,37 @@ export class Storage extends Service {
       }
     );
   }
+
+  /**
+   * Get a reference to an HmacKey object.
+   * Note: this does not fetch the HMAC key's metadata. Use HmacKey#get() to
+   * retrieve and populate the metadata.
+   *
+   * @param {string} accessId The HMAC key's access ID.
+   * @returns {HmacKey}
+   * @see HmacKey
+   *
+   * @example
+   * const {Storage} = require('@google-cloud/storage');
+   * const storage = new Storage();
+   * const hmacKey = storage.hmacKey('ACCESS_ID');
+   */
+  hmacKey(accessId: string) {
+    return new HmacKey(this, accessId);
```

Comment:
We should throw here if accessId is not provided, and remove the throw from the HmacKey constructor.

Reply:
fixed